### DISCUSSION TITLE
Does SuiteAdvanced Run on Edge?

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#137] [Q&A] [answered]
Wondering if SuiteAdvanced runs on Edge.  It's been my experience (YMMV) that NetSuite in general runs faster on Edge.

**Comment by burkybang** [comment, marked as answer]
Yes, SuiteAdvanced should work on all Chromium-based browsers.

**Reply by dd-manderson** [reply]
Where would I find it?

I don't see it here:
https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home

**Reply by dd-manderson** [reply]
nm -- answer here

https://support.microsoft.com/en-us/microsoft-edge/add-turn-off-or-remove-extensions-in-microsoft-edge-9c0ec68c-2fbc-2f2c-9ff0-bdc76f46b026#:~:text=Add%20an%20extension%20to%20Microsoft%20Edge%20from%20the%20Chrome%20Web,then%20select%20Allow%20to%20confirm.

**Reply by burkybang** [reply]
Yeah, you just go to the Chrome Web Store in Edge. You can also go to [SuiteAdvanced.com](https://SuiteAdvanced.com) and click the Download button.
